compose-shadow/src/commonMain/kotlin/com/adamglin/composeshadow/utils/BlendMode.kt

```diff
@@ -0,0 +1,24 @@
+package com.adamglin.composeshadow.utils
+
+enum class BlendMode(
```

Comment:
Would it be better to use ` androidx.compose.ui.graphics.BlendMode` instead?

And add `fun BlendMode.toSkia()` and fun `BlendMode.toPorterDuffMode()`

Reply:
- Since there was no way to use androidx.compose.ui.graphics.BlendMode in NativePaint, it was changed to use org.jetbrains.skia.BlendMode instead.

- Additionally, since PorterDuff couldn't be moved to commonMain, we modified the implementation to include BlendMode.toPorterDuffMode within the Android-specific code

Reply:
You are correct. However, I believe that incorporating Skia into the Android section is not appropriate. Based on your code, I suggest the following structure:

1. In `commonMain`:

```kotlin
internal expect fun NativePaint.setBlendMode(blendMode: androidx.compose.ui.graphics.BlendMode)
```

2. In `androidMain`:

```kotlin
internal actual fun NativePaint.setBlendMode(blendMode: BlendMode) {
    xfermode = PorterDuffXfermode(blendMode.toPorterDuffMode())
}

private fun BlendMode.toPorterDuffMode(): PorterDuff.Mode = ...
```

3. In `appleMain`:

```kotlin
internal actual fun NativePaint.setBlendMode(blendMode: BlendMode) {
    xfermode = PorterDuffXfermode(blendMode.toSkia())
}

private fun BlendMode.toSkia(): PorterDuff.Mode = ...
```

Reply:
If you check the newly committed changes below, you can see that it has been implemented in a similar structure

1. In `commonMain`,`NativePaint.desktop.kt`
```kotlin
internal expect fun NativePaint.setBlendMode(blendMode: BlendMode)
```

2. In `androidMain`

```kotlin 
// NativePaint.android.kt
internal actual fun NativePaint.setBlendMode(blendMode: BlendMode) {
    xfermode = PorterDuffXfermode(blendMode.toPorterDuffMode())
}
```
```kotlin
// BlendMode.kt
fun BlendMode.toPorterDuffMode(): PorterDuff.Mode {
    return when (this) {
        // ...
    }
}
```

3. In `appleMain`
```kotlin
internal actual fun NativePaint.setBlendMode(blendMode: BlendMode) {
    this.blendMode = blendMode
}
```